R/visr.R

```diff
@@ -235,11 +235,18 @@ visr.survfit <- function(
 
 # Obtain X-asis label ----------------------------------------------------------
 
-  if (is.null(x_label)){
+  if (is.null(x_label)) {
+    
     if ("PARAM" %in% names(x)) x_label = x[["PARAM"]]
     if (! "PARAM" %in% names(x)) x_label = "time"
```

Comment:
Then x_label = what the user specifies. Don't force "time".

Reply:
Well, I'm not :D 

If `x_label` and `x_units` are both not `NULL`, they are concatenated to `<x_label> (<x_units>)`.

If `x_label` is not `NULL`, but `x_units` is it set's the label to `x_label`.

If `x_label` is `NULL`, but `x_units` is not, it tries to deduct the name from the data. If that is not possible, it's set to the default "time". And then it appends `x_units`.

If `x_label` and `x_units` are `NULL`, it tries to deduct the name from the data. If that is not possible, it's set to the default "time".

Reply:
Dont use "time" as default. Default is NULL.